CONTRIBUTING.md

```diff
@@ -1,70 +1,120 @@
-Contributing to Calypso
-=======================
+# Contributing to Calypso
 
-Calypso is Open Source software licensed under [GNU General Public License v2 (or later)](./LICENSE.md).
```

Comment:
I think we should have a ref to the GPL - but can also mention it in a friendly way...maybe:

"Calypso is released under GPL v2 (or later) - the same license as WordPress. We chose the GPL to help make sure that Calypso's code is free, and available for anyone to use and build upon. In the spirit of the GPL - we encourage you to make your own contribution and are excited to see what we can build together"